js/capture.js

```diff
@@ -3,8 +3,16 @@
 * third-party requests to storage.
 */
 const capture = {
-  init() {
+  async init() {
     this.addListeners();
```

Comment:
The other solution to the whitelist not working is move this after the whitelist has loaded. This means we shouldn't block capturing by the entity list.

If the fetch fails perhaps make the lists empty arrays?